packages/messenger-widget/src/worker.js

```diff
@@ -0,0 +1,15 @@
+import logo from './assets/images/dm3.svg';
+
+// service worker to detect any push notification is triggered
+self.addEventListener('push', (ev) => {
```

Comment:
is it possible to change the event name to dm3_register_push_notification to have the namespace clear?

Reply:
In the documentation, I can't see anywhere that the event name can be changed. So we have to remain stick with this.